user-restricted-separate-auth-tutorials/cis2/csharp/HelloWorldSeparateAuth/Program.cs

```diff
@@ -0,0 +1,23 @@
+namespace HelloWorldSeparateAuth;
```

Comment:
This entire file is unused.

Reply:
Program.cs is the default entry point for dotnet applications, it's also used to make the access token accessible to multiple namespaces via the `Config` class